submissions/adeyinkaQuadri/css/foodbag.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/foodbag.css">
+    <title>FoodBag</title>
+</head>
+<body>
+    <nav class="nav">
+     <div class="nav-logo" ><h2>Food<span>Bag</span></h2></div>   
+        <ul class="nav-list">
+        <a href=""><li>cart</li></a>    
+         <a href="" ><li>login</li></a>   
+           <a href=""><li><input type="button" value="Sign up"></li></a> 
+        </ul>
+    </nav>
+    <!-- section -->
+
+    <section>
+<div class="foodbag-motto">
+    <h1>FoodBag</h1>
+    <p>Anything food, we have you covered</p>
+</div>
+    </section>
+
+    <section>
+        <div class="location"><h4>Where are you?</h4>
+            <br>
+        <input type="text" name="" id="" list="dagger" value="Bodija Market">
+        <datalist id="dagger">
+            <option value="Bodija Market"></option>
+            <option value="Sodija Market"></option>
+        </datalist>
+        </div>
+    </section>
+<!--section 2-->
+    <section>
+        <div class="intrest"><h4>pick an interest</h4>
+            <div class="interest-container">
+                <div class="body1">
+                    <div class="img-cover"><img src="img/fast-food.svg" alt=""></div>
+                    <div class="text-cover">
+                        <h4>Fast food</h4>
+                        <p>All sorrows are less with bread</p>
+                    </div>
+                </div>
+                <div class="body1">
+                    <div class="img-cover"><img src="img/Vegetables & Drinks.svg" alt=""></div>
+                    <div class="text-cover">
+                        <h4>Vegetables & Fruits</h4>
+                        <p>May not want it, but good for you</p>
+                    </div>
+                </div>
+                <div class="body1">
+                    <div class="img-cover"><img src="img/Drinks & Cocktails.svg" alt=""></div>
+                    <div class="text-cover">
+                        <h4>Drink & Cocktails</h4>
+                        <p>I feel sad for those who dont drink</p>
+                    </div>
+                </div>
+                <div class="body1">
+                    <div class="img-cover"><img src="img/resturant.svg" alt=""></div>
+                    <div class="text-cover">
+                        <h4>Resturants</h4>
+                        <p>All sorrows are less with bread</p>
+                    </div>
+                </div>
+
+<div class="line" >
+
+</div>
+
+            </div>
+        
+        </div>
+    </section>
+
+    <!--another section-->
+
+    <section>
+        <h2 style="text-align:center;color:#4e4d4d;font-size:23px;font-weight:600;">How we serve you</h2>
```

Comment:
Remove the style here, its bad coding practice, name it and select it in the css file.